packages/portalnetwork/src/networks/network.ts

```diff
@@ -288,19 +364,53 @@ export abstract class BaseNetwork extends EventEmitter {
       // Check to see if node is already in corresponding network routing table and add if not
       const enr = this.findEnr(src.nodeId)
       if (enr !== undefined) {
-        this.updateRoutingTable(enr, pingMessage.customPayload)
+        this.updateRoutingTable(enr)
       }
-    } else {
-      const radius = PingPongCustomDataType.deserialize(pingMessage.customPayload).radius
-      this.routingTable.updateRadius(src.nodeId, radius)
     }
-    await this.sendPong(src, id)
+    const { type, payload } = CustomPayloadExtensionsFormat.deserialize(pingMessage.customPayload)
+    let pongPayload: Uint8Array
+    switch (type) {
+      case PingPongPayloadExtensions.CLIENT_INFO_RADIUS_AND_CAPABILITIES: {
+        pongPayload = this.pingPongPayload(type)
+        break
+      }
+      case PingPongPayloadExtensions.BASIC_RADIUS_PAYLOAD: {
+        if (this.networkId !== NetworkId.HistoryNetwork) {
```

Comment:
Shouldn't we support basic radius on all networks?

Reply:
Ah, whoops.  that should be in the HISTORY_RADIUS case